plugins/github-actions/src/components/WorkflowRunsCard/WorkflowRunsCard.tsx

```diff
@@ -0,0 +1,416 @@
+                          run.conclusion as keyof typeof statusColors
+                        ] as Color
+                      }
+                      style={{ alignItems: 'center' }}
+                    >
+                      <Typography variant="h6">
+                        <Link
+                          component={RouterLink}
+                          to={routeLink({ id: run.id })}
+                        >
+                          <Typography color="primary" variant="h6">
+                            {run.workflowName}
+                          </Typography>
+                        </Link>
+                      </Typography>
+                    </Alert>
+                  </Tooltip>
+                  <Tooltip title={run.message ?? 'No run message'}>
+                    <Typography
+                      variant="body2"
+                      component="span"
+                      style={{ fontSize: 'smaller' }}
+                    >
+                      <MarkdownContent
+                        content={`Commit ID : ${run.source.commit.hash!}`}
+                      />
+                    </Typography>
+                  </Tooltip>
+
+                  {run.source.branchName && (
+                    <MarkdownContent
+                      content={`Branch : ${run.source.branchName}`}
+                    />
+                  )}
+                  <Chip
+                    key={run.id}
+                    size="small"
+                    label={`Workflow ID : ${run.id}`}
+                  />
+                  <Chip
+                    size="small"
+                    label={
+                      <Box display="flex" alignItems="center">
+                        <WorkflowRunStatus
+                          status={run.status}
+                          conclusion={run.conclusion}
+                        />
+                      </Box>
+                    }
+                  />
+                  <div className={classes.bottomline}>
+                    {run.githubUrl && (
+                      <Link to={run.githubUrl}>
+                        Workflow runs on GitHub{' '}
+                        <ExternalLinkIcon
+                          className={classes.externalLinkIcon}
+                        />
+                      </Link>
+                    )}
+                    <ButtonGroup>
+                      <Tooltip title="Rerun workflow">
+                        <IconButton
+                          onClick={run.onReRunClick}
+                          style={{ fontSize: '12px' }}
+                        >
+                          <RetryIcon />
+                        </IconButton>
+                      </Tooltip>
+                    </ButtonGroup>
+                  </div>
+                </Box>
+              </Box>
+            </Box>
+          </Grid>
+        ))
+      ) : (
+        <Box p={2}>
+          {loading ? <CircularProgress /> : 'No matching runs found.'}
```

Comment:
We have been pretty adamant about using `Progress` from `@backstage/core-components`, but ... in a card context like this, I guess this may be an ok fit